package.json

```diff
@@ -91,5 +91,8 @@
       "vite": "workspace:*",
       "@vitejs/plugin-vue": "workspace:*"
     }
+  },
+  "dependencies": {
+    "compress": "^0.99.0"
   }
```

Comment:
Is this dependency needed in the root package.json?

Reply:
there's two tests which uses compression.
I can't use the modern compression version without exporting the function

Reply:
I see `ssr-react` and `ssr-vue` playgrounds using the `compression` package and have it specified under their `devDependencies` respectively. This `compress` dependency doesn't seem to be used anywhere, perhaps I'm missing something else? 🤔

Reply:
yes. it's those playgrounds. I didn't notice they had them as dev dep